crates/next-api/src/module_graph.rs

```
@@ -509,6 +502,18 @@ async fn validate_pages_css_imports(
         Either::Right(graph.entry_modules())
     };
 
+    // We need to collect the module names here, the traverse_edges_from_entries callback cannot
+    // await futures.
+    let module_ident_map = graph
+        .graph
+        .node_weights()
+        .map(async |n| Ok((n.module(), n.module().ident().to_string().await?)))
+        .try_join()
+        .await
```

is this implicitly ignoring an Error result?

---

Nice catch, I copied that from somewhere else 😬